en-US/Audience.xml

```diff
@@ -6,11 +6,15 @@
 <section id="audience">
 	<title>Audience</title>
 	 <para>
-		This guide is the official style guide for technical documentation for Red&nbsp;Hat training and certification content, including how to write for global audiences and translation, common mistakes to avoid, rules for everyday punctuation, and grammar.
+		This guide is the official style guide for Red&nbsp;Hat training and certification content, and for all other technical documentation except as stated.
+		The guide includes everyday punctuation and grammar, common mistakes to avoid, strategies for translation and global audiences, and a word usage dictionary.
 	</para>
 	 <para>
-        Other Red&nbsp;Hat technical documentation, including product documentation by Customer Content Services (CCS), follows the Red&nbsp;Hat supplementary style guide for product documentation at <ulink url="https://redhat-documentation.github.io/supplementary-style-guide/" /> instead of this Red&nbsp;Hat Technical Writing Style Guide.
+        Red&nbsp;Hat product documentation by Customer Content Services (CCS) follows the Red&nbsp;Hat supplementary style guide at <ulink url="https://redhat-documentation.github.io/supplementary-style-guide/" /> instead of this Red&nbsp;Hat Technical Writing Style Guide.
```

Comment:
Consider moving line 13 up to follow line 9. Line 9 now ends with "and for all other technical documentation except as stated", which primes readers to look for the exceptions (which line 13 delivers). This way, paragraph 1 would be all about the audience (who is using which guide). 

If line 13 moves up, I suggest creating a new paragraph for the current line 10 ("This guide includes..."), since it presents a new topic: the guide's purpose (by way of key topic areas).